packages/tx-construction/src/tx-builder/types.ts

```diff
@@ -11,7 +11,10 @@ import { OutputBuilderValidator } from './OutputBuilder';
 import { OutputValidation } from '../output-validation';
 
 export type PartialTxOut = Partial<
-  Pick<Cardano.TxOut, 'address' | 'datumHash' | 'datum' | 'scriptReference'> & { value: Partial<Cardano.Value> }
+  Pick<Cardano.TxOut, 'address' | 'datumHash' | 'datum' | 'scriptReference'> & {
+    value: Partial<Cardano.Value>;
+    handle: Handle;
```

Comment:
Not all TxOut have associated handles, especially not this partial type
```suggestion
    handle?: Handle;
```